packages/svelte/src/compiler/migrate/index.js

```diff
@@ -331,7 +331,10 @@ const instance_script = {
 
 				const binding = /** @type {Compiler.Binding} */ (state.scope.get(declarator.id.name));
 
-				if (state.analysis.uses_props && (declarator.init || binding.updated)) {
+				if (
+					state.analysis.uses_props &&
+					(declarator.init || binding.mutated || binding.reassigned)
```

Comment:
why did this change? `binding.updated` already means `binding.mutated || binding.reassigned`